### ISSUE TITLE
lint for detecting objects with too many attributes

### ISSUE THREAD

**Issue by ext11n7** [objectionary/lints#996]
## Problem

Currently, there is no lint that warns when an EO object declares too many attributes. Objects with excessive attributes are hard to read, test, and reuse. This violates the principle of keeping objects small and focused.

## Suggestion

Add a new lint `too-many-attributes` that reports a defect when an object declares more than a configurable threshold of attributes (e.g., 5 by default).

## Example of code that should trigger the lint

```
[a b c d e f g] +> my-object
  ...
```

This object has 7 attributes, which is too many.

## Expected behavior

The lint should report a `WARNING` level defect pointing to the object and suggesting to split it into smaller objects.

**Comment by ext11n7** [comment]
@yegor256

**Comment by yegor256** [comment]
@ext11n7 I believe we already have such a lint. @volodya-lombrozo ?

**Comment by volodya-lombrozo** [comment]
@yegor256 @ext11n7 Sure, we already have exactly this lint:
https://github.com/objectionary/lints/blob/master/src/main/resources/org/eolang/motives/errors/many-void-attributes.md

@ext11n7 Please, fill free to create a new issue if there is some misunderstanding from my side

**Comment by 0crat** [comment]
@ext11n7 Thanks for submitting this ticket! However, as you see, it was not accepted by the team, which is sad. You've earned -8 points. By reporting bugs or suggesting enhancements, you help our project improve its quality. Please, don't be discouraged. Instead, try to be more careful next time, and [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) bugs. Your running score is [-8](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
